kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/kroxylicious/Kroxylicious.java

```diff
@@ -119,4 +122,17 @@ public String getBootstrap() {
         LOGGER.debug("Kroxylicious bootstrap: {}", bootstrap);
         return bootstrap;
     }
+
+    /**
+     * Scale replicas to.
+     *
+     * @param scaledTo the number of replicas to scale up/down
+     * @param timeout the timeout
+     */
+    public void scaleReplicasTo(int scaledTo, Duration timeout) {
+        LOGGER.info("Scaling number of replicas to {}..", scaledTo);
+        kubeClient().getClient().apps().deployments().inNamespace(deploymentNamespace).withName(Constants.KROXY_DEPLOYMENT_NAME).scale(scaledTo);
+        await().atMost(timeout).pollInterval(Duration.ofSeconds(5))
```

Comment:
nit: `5s` seems like quite a slow poll interval (especially for running locally).